docs/features/sharepoint.md

```diff
@@ -0,0 +1,77 @@
+# Sharepoint File Ingestion
```

Comment:
Sharepoint is a product so spelling matters.. please ensure we are using "SharePoint" in all places. The S & P should both be capitalized.

Reply:
Fixed